src/pb_stub.cc

```diff
@@ -459,6 +437,12 @@ Stub::AutoCompleteModelConfig(
   if (py::hasattr(sys.attr("TritonPythonModel"), "auto_complete_config")) {
     model_config = sys.attr("TritonPythonModel")
                        .attr("auto_complete_config")(model_config);
+  } else if (py::hasattr(
+                 sys.attr("TritonPythonModel"),
+                 "plugin_auto_complete_config")) {
```

Comment:
I don't think we need a new API for these models. Let's just call the regular Python model APIs.

Reply:
The new APIs were added to additionally provide the complete path to the framework model. In this case the model.saved_model file living within model-repository.
Modifying the current function signature would not be backwards-compatible and all the current python models might start failings.
I think we can get by using a single initialize function as it consumes generic args, but for auto-complete-config, I don't know how we can retrieve the model file path.